xml_inscriptions/bib_only/MA.Bos.MFA.G.01.7479.xml

```diff
@@ -66,14 +131,30 @@
       <revisionDesc>
          <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-06-22" who="Jean Wanlass">expansion of bib only stub</change>
       </revisionDesc>
    </teiHeader>
-   <!--facsimile element goes here-->
+   
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc>photo: MFA 01.7479</desc>
+         <graphic url="MA.Bos.MFA.G.01.7479.jpg"/>
+      </surface>
+   </facsimile>
+   
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+            <ab>ΤΟDΙΟΣΟΛΥΝΠΙΟ</ab></div>
+         
+         <div type="translation">
+            <p>Of Olympian Zeus</p></div>
+         
+         <div type="commentary">
+            <p>in retrograde</p></div>
```

Comment:
You should be able to indicate this in the text area as well, on either the `<div>` (if it applies to the whole text) or the `<lb>` if just a single line; see https://epidoc.stoa.org/gl/latest/trans-linebreakdirection.html.